modules/effects/src/effect_creator.ts

```diff
@@ -10,6 +10,15 @@ import {
 
 type DispatchType<T> = T extends { dispatch: infer U } ? U : true;
 type ObservableType<T, OriginalType> = T extends false ? OriginalType : Action;
+type EffectResult<OT> = Observable<OT> | ((...args: any[]) => Observable<OT>);
+type ConditionallyDisallowActionCreator<DT, Result> = DT extends false
+  ? unknown // If DT (DispatchType is false, then we don't enforce any return types)
+  : Result extends EffectResult<infer OT>
+    ? OT extends ActionCreator
+      ? 'ActionCreator cannot be dispatched. Did you forget to call action creator function?'
```

Comment:
```suggestion
      ? 'ActionCreator cannot be dispatched. Did you forget to call the action creator function?'
```

Reply:
I was going for the similar message as we have here: https://github.com/ngrx/platform/blob/6caae70677166e24468f9d69807e3efe0d95bee3/modules/store/src/store.ts#L101

Should I adjust `the` there as well?

Reply:
I think so, I think that's how it's written?  🤷‍♂️

Reply:
Thanks Tim! Adjusted in both places.